projects/dashboards/gfbio/src/app/app.component.html

```diff
@@ -65,6 +65,10 @@
             </ng-container>
         </geoengine-map-container>
     </div>
+    <geoengine-map-info
+        *ngIf="config.COMPONENTS.MAP_INFO.AVAILABLE"
```

Comment:
I wasn't really involved in the design process of this, if there was any. But: isn't this a bit too prominently places? it takes a lot of space that covers the map.

Can we not instead put this info in the top bar? We also show the selected time there, so I think it would make sense to show the bbox there as well, as both make up the query rectangle.